### ISSUE TITLE
Problem on RPi4 (64 bit)

### ISSUE THREAD

**Issue by edwardreed81** [jacobalberty/unifi-docker#311]
### Host operating system
Ubuntu server 20.04 64 bit (Raspberry Pi 4 image)

### What tag are you using
arm32v7

### What complete docker command or docker-compose.yml do you use to launch the container (omitting sensitive values)?
docker-compose:

```
version: '3.7'

networks:
  unifi:
  reverse_proxy:
    external: true

services:
  mongo:
    image: mongo:3.4
    networks:
      - unifi
    restart: unless-stopped
    volumes:
      - ./data/db:/data/db
  controller:
    image: jacobalberty/unifi:arm32v7
    depends_on:
      - mongo
    init: true
    networks:
      - unifi
      - reverse_proxy
    restart: unless-stopped
    volumes:
      - ./unifi/data:/unifi/data
      - ./unifi/log:/unifi/log
      - ./unifi/cert:/unifi/cert
      - ./unifi/init:/unifi/init.d
    environment:
      DB_URI: mongodb://mongo/unifi
      STATDB_URI: mongodb://mongo/unifi_stat
      DB_NAME: unifi
      RUNAS_UID0: "false"
      UNIFI_UID: 1000
      UNIFI_GID: 1000
      TZ: America/New_York
    ports:
      - "3478:3478/udp" # STUN
      - "6789:6789/tcp" # Speed test
      - "8080:8080/tcp" # Device/ controller comm.
      - "8443:8443/tcp" # Controller GUI/API as seen in a web browser
      - "8880:8880/tcp" # HTTP portal redirection
      - "8843:8843/tcp" # HTTPS portal redirection
      - "10001:10001/udp" # AP discovery
```

### What do you expect to happen?

Normal startup

### What actually happens?
```
mongo_1       | 2020-06-01T16:53:39.789+0000 I CONTROL  [initandlisten] MongoDB starting : pid=1 port=27017 dbpath=/data/db 64-bit host=ee688a51a601
mongo_1       | 2020-06-01T16:53:39.791+0000 I CONTROL  [initandlisten] db version v3.4.24
mongo_1       | 2020-06-01T16:53:39.791+0000 I CONTROL  [initandlisten] git version: 865b4f6a96d0f5425e39a18337105f33e8db504d
mongo_1       | 2020-06-01T16:53:39.791+0000 I CONTROL  [initandlisten] OpenSSL version: OpenSSL 1.0.2g  1 Mar 2016
mongo_1       | 2020-06-01T16:53:39.791+0000 I CONTROL  [initandlisten] allocator: tcmalloc
mongo_1       | 2020-06-01T16:53:39.791+0000 I CONTROL  [initandlisten] modules: none
mongo_1       | 2020-06-01T16:53:39.791+0000 I CONTROL  [initandlisten] build environment:
mongo_1       | 2020-06-01T16:53:39.791+0000 I CONTROL  [initandlisten]     distmod: ubuntu1604
mongo_1       | 2020-06-01T16:53:39.791+0000 I CONTROL  [initandlisten]     distarch: aarch64
mongo_1       | 2020-06-01T16:53:39.791+0000 I CONTROL  [initandlisten]     target_arch: aarch64
mongo_1       | 2020-06-01T16:53:39.791+0000 I CONTROL  [initandlisten] options: {}
mongo_1       | 2020-06-01T16:53:39.791+0000 I -        [initandlisten] Assertion failure getrlimit(RLIMIT_NOFILE, &limit) == 0 src/mongo/util/net/listen.cpp 624
mongo_1       | 2020-06-01T16:53:39.828+0000 I CONTROL  [initandlisten]
mongo_1       |  0xaaaadd042150 0xaaaadcfd8024 0xaaaadc401678 0xaaaadcfe8540 0xaaaadcfe8604 0xaaaadcfea380 0xaaaadcfb6340 0xaaaadc3eac0c 0xaaaadc40b4ac 0xffffafb988a0
mongo_1       | ----- BEGIN BACKTRACE -----
mongo_1       | {"backtrace":[{"b":"AAAADBBFB000","o":"1447150","s":"_ZN5mongo15printStackTraceERSo"},{"b":"AAAADBBFB000","o":"13DD024","s":"_ZN5mongo10logContextEPKc"},{"b":"AAAADBBFB000","o":"806678","s":"_ZN5mongo12verifyFailedEPKcS1_j"},{"b":"AAAADBBFB000","o":"13ED540","s":"_ZN5mongo17getMaxConnectionsEv"},{"b":"AAAADBBFB000","o":"13ED604","s":"_ZN5mongo8Listener18checkTicketNumbersEv"},{"b":"AAAADBBFB000","o":"13EF380","s":"_ZN5mongo8Listener12setupSocketsEv"},{"b":"AAAADBBFB000","o":"13BB340","s":"_ZN5mongo9transport20TransportLayerLegacy5setupEv"},{"b":"AAAADBBFB000","o":"7EFC0C"},{"b":"AAAADBBFB000","o":"8104AC","s":"main"},{"b":"FFFFAFB79000","o":"1F8A0","s":"__libc_start_main"}],"processInfo":{ "mongodbVersion" : "3.4.24", "gitVersion" : "865b4f6a96d0f5425e39a18337105f33e8db504d", "compiledModules" : [], "uname" : { "sysname" : "Linux", "release" : "5.4.0-1011-raspi", "version" : "#11-Ubuntu SMP Fri May 8 07:43:33 UTC 2020", "machine" : "aarch64" }, "somap" : [ { "b" : "AAAADBBFB000", "elfType" : 3, "buildId" : "3DBB9C4940ED3315B418BA05231DE10CF0E5E6F9" }, { "b" : "FFFFB001E000", "elfType" : 3, "buildId" : "EBD93BAD5C6559B2D840593F079A2004E31AA41D" }, { "b" : "FFFFAFF89000", "path" : "/lib/aarch64-linux-gnu/libssl.so.1.0.0", "elfType" : 3, "buildId" : "124D415B526BD3F23D860A22D61E1BB2E78225DB" }, { "b" : "FFFFAFDE4000", "path" : "/lib/aarch64-linux-gnu/libcrypto.so.1.0.0", "elfType" : 3, "buildId" : "57D86F63CECE2854EEB0BCB2902539F32FF8B74D" }, { "b" : "FFFFAFDCD000", "path" : "/lib/aarch64-linux-gnu/librt.so.1", "elfType" : 3, "buildId" : "E29A22F305FD4ED854F285075E0FCF108214FB05" }, { "b" : "FFFFAFDBA000", "path" : "/lib/aarch64-linux-gnu/libdl.so.2", "elfType" : 3, "buildId" : "24F2003BAAC8931062D7877508ACF12C16F043BA" }, { "b" : "FFFFAFD0D000", "path" : "/lib/aarch64-linux-gnu/libm.so.6", "elfType" : 3, "buildId" : "80D706B4DF38445B5D79F6D178E22119A310BE81" }, { "b" : "FFFFAFCEC000", "path" : "/lib/aarch64-linux-gnu/libgcc_s.so.1", "elfType" : 3, "buildId" : "A05222936FB1A047AC0567B0861730C781C69A23" }, { "b" : "FFFFAFCC0000", "path" : "/lib/aarch64-linux-gnu/libpthread.so.0", "elfType" : 3, "buildId" : "5B6DA4B4C11C0FD847E788E610B22C253C8BF1EA" }, { "b" : "FFFFAFB79000", "path" : "/lib/aarch64-linux-gnu/libc.so.6", "elfType" : 3, "buildId" : "825D6B66F1FFDA9DF55C4D8EC221D57D8BCC82A4" }, { "b" : "FFFFAFFF3000", "path" : "/lib/ld-linux-aarch64.so.1", "elfType" : 3, "buildId" : "18CCB66DA0CF21AB4C1B1FE62B8D01C405A8825A" } ] }}
mongo_1       |  mongod(_ZN5mongo15printStackTraceERSo+0x48) [0xaaaadd042150]
mongo_1       |  mongod(_ZN5mongo10logContextEPKc+0x104) [0xaaaadcfd8024]
mongo_1       |  mongod(_ZN5mongo12verifyFailedEPKcS1_j+0x154) [0xaaaadc401678]
mongo_1       |  mongod(_ZN5mongo17getMaxConnectionsEv+0x190) [0xaaaadcfe8540]
mongo_1       |  mongod(_ZN5mongo8Listener18checkTicketNumbersEv+0x34) [0xaaaadcfe8604]
mongo_1       |  mongod(_ZN5mongo8Listener12setupSocketsEv+0x40) [0xaaaadcfea380]
mongo_1       |  mongod(_ZN5mongo9transport20TransportLayerLegacy5setupEv+0x30) [0xaaaadcfb6340]
mongo_1       |  mongod(+0x7EFC0C) [0xaaaadc3eac0c]
mongo_1       |  mongod(main+0x794) [0xaaaadc40b4ac]
mongo_1       |  libc.so.6(__libc_start_main+0xE0) [0xffffafb988a0]
mongo_1       | -----  END BACKTRACE  -----
mongo_1       | 2020-06-01T16:53:39.841+0000 I STORAGE  [initandlisten] exception in initAndListen: 0 assertion src/mongo/util/net/listen.cpp:624, terminating
mongo_1       | 2020-06-01T16:53:39.841+0000 I NETWORK  [initandlisten] shutdown: going to close listening sockets...
mongo_1       | 2020-06-01T16:53:39.841+0000 I NETWORK  [initandlisten] shutdown: going to flush diaglog...
mongo_1       | 2020-06-01T16:53:39.841+0000 I CONTROL  [initandlisten] now exiting
mongo_1       | 2020-06-01T16:53:39.841+0000 I CONTROL  [initandlisten] shutting down with code:100
```

I realize that this isn't really an issue with this image, but rather with the MongoDB image, but I didn't know where else to turn for help.

I've been running this image on a amd64 machine for a long time and just recently switched to a Raspberry Pi 4. It seems like there's a MongoDB 3.4 image for this arch, but it doesn't start properly, giving me the error above. My Googling has been fruitless and I was hoping someone here might be able to shed some light on what is going on.

**Comment by 3DIRK** [comment]
I had the same problem. Switched to Mongo Image version 3.6 and everything worked fine.

**Comment by edwardreed81** [comment]
I tried Mongo 3.6 as well and got similarly poor results. Strange that everyone is experiencing different things.

**Comment by jgorinsky** [comment]
I ran into this same problem running Ubuntu 20.04 on an Rpi4.

I'm thinking it is probably https://github.com/seccomp/libseccomp/issues/234, which is due to be fixed in libseccomp-2.4.4.

**Comment by jgorinsky** [comment]
Installing libseccomp 2.4.3-1ubuntu3.16.04.2 from the focal-proposed repo did clear up the problem for me. I'm not familiar enough with Ubuntu's release processes to know when that will land in the release repo though.
